k8_cortx_cloud/deploy-cortx-cloud.sh

```diff
@@ -328,6 +328,12 @@ buildValues() {
     set +eu
 }
 
+if [[ -n $(helm ls --all-namespaces --short -f '^cortx$') ]]; then
```

Comment:
```suggestion
if [[ -n $(helm ls --all-namespaces --short --filter '^cortx$') ]]; then
```

My fault for the original suggestion, can you please use the long option?